packages/hooks/src/index.ts

```diff
@@ -5,5 +5,6 @@ export { default as useRowSelection } from './use-row-selection';
 export { default as useSorting } from './use-sorting';
 export { default as usePaginationState } from './use-pagination-state';
 export { default as useDataTableSortingState } from './use-data-table-sorting-state';
+export * from './use-data-table-sorting-state';
```

Comment:
Hmm I would be more explicit maybe if this is intended to export the types

```suggestion
export * from './use-data-table-sorting-state/export-types';
```

Reply:
Thanks for the suggestions.

Updated with this commit: b31956abfaa3855ece61e57a532205774607271d